treeherder/perf/alerts.py

```diff
@@ -15,6 +17,51 @@
                                             detect_changes)
 
 
+def check_confirming_perf_alerts_status():
+    confirming_interval = settings.PERFHERDER_CONFIRMING_INTERVAL
+
+    def traceback_producer_job_type(perf_alert):
+        push = perf_alert.summary.push
+        signature = perf_alert.series_signature
+
+        associated_perf_datum = PerformanceDatum.objects.filter(push=push, signature=signature).first()
+        return associated_perf_datum.job.job_type if associated_perf_datum else None
+
+    def extract_pushes_to_check(many_nearby_pushes, alert_push, push_range=2):
+        many_nearby_pushes = list(many_nearby_pushes)
+
+        alert_push_idx = None
+        for index, push in enumerate(many_nearby_pushes):
+            if alert_push.id == push.id:
+                alert_push_idx = index
+                break
+
+        from_push = alert_push_idx - push_range
+        from_push = from_push if from_push > 0 else 0
+        to_push = alert_push_idx + push_range
```

Comment:
while we care about +-2 for a push range, we really care about +-2 existing data points- the difference is that push range can include pushes that have no data (i.e. needs a backfill.)  In this case we would need a larger range to consider.  Also in many cases we cannot backfill (broken builds, infra down, etc.) and this +-2 is going to fail and be confusing as our edge cases will pile up.